solace-java-spring-boot-autoconfigure/src/main/java/com/solace/labs/spring/boot/autoconfigure/SolaceJavaProperties.java

```diff
@@ -66,6 +66,31 @@
      */
     private String clientName;
     
+    /**
+     * The number of times to attempt and retry a connection to the host Solace Message Router (or list of routers) during initial connection setup.
+     */
+    private int connectRetries = 0;
+    
+    /**
+     * The number of times to attempt to reconnect to the host Solace Message Router (or list of routers) after an initial connected session goes down.
+     */
+    private int reconnectRetries = 3;
+    
+    /**
+     * When using a host list for the HOST property, this property defines how many times to try 
+     * to connect or reconnect to a single host before moving to the next host in the list. 
+     * NOTE: This property works in conjunction with the connect and reconnect retries settings; 
+     * it does not replace them.
+     */
+    private int connectRetriesPerHost = 0; 
+    
+    /**
+     * How much time in (MS) to wait between each attempt to connect or reconnect to a host. 
+     * If a connect or reconnect attempt to host is not successful, the API waits for the 
+     * amount of time set for reconnectRetryWaitInMillis, and then makes another connect or 
+     * reconnect attempt.
+     */
+    private int reconnectRetryWaitInMillis = 3000;
     
```

Comment:
I think that is a good idea. Will update to the recommended value and also update the unit test.